src/Pdk/Hooks/PdkAdminEndpointHooks.php

```diff
@@ -36,6 +36,14 @@ public function registerPdkRoutes(): void
             'methods'             => WP_REST_Server::ALLMETHODS,
             'callback'            => [$this, 'processBackendRequest'],
             'permission_callback' => function () {
+                if (! is_user_logged_in()) {
+                    return false;
+                }
+                
+                if ('shop_manager' === (wp_get_current_user()->roles[0] ?? '')) {
```

Comment:
dit kan beter in de pdk config, zodat het makkelijk te vinden en configureren is